packages/vscode-extension/src/plugins/network/strategies/DebuggerNetworkInspector.ts

```diff
@@ -249,15 +252,8 @@ export default class DebuggerNetworkInspector extends BaseNetworkInspector {
     this.activationState = ActivationState.Inactive;
   }
 
-  public deactivate(): void {
-    this.cleanup(true);
-  }
-
-  /**
-   * Suspends without clearing messages to preserve state across reactivation
-   */
-  public suspend(): void {
-    this.cleanup(false);
+  public disable(): void {
+    this.deactivate(true);
```

Comment:
Isn't this the only place you pass `true` here? I think it'd be cleaner to have the `disable`-specific login inside the `disable` method instead of bolting an extra parameter to `deactivate`
```suggestion
    this.deactivate();
    this.networkBridge.disableNetworkInspector();
    this.clearNetworkMessages();
```
Same goes for the other `NetworkInspector` impl.